src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -176,19 +194,22 @@ def _get_native_state(self):
         return state
 
     def _get_version(self):
-        return 1
+        return self._PYTHON_DRAWING_CLASSIFIER_VERSION
 
     @classmethod
     def _load_version(cls, state, version):
-        _tkutl._model_version_check(version, 1)
+        _tkutl._model_version_check(version, 
+            cls._PYTHON_DRAWING_CLASSIFIER_VERSION)
         from ._model_architecture import Model as _Model
-        net = _Model(num_classes = len(state['classes']), prefix = 'model0_')
+        net = _Model(num_classes = len(state['classes']), prefix = 'drawing_')
         ctx = _mxnet_utils.get_mxnet_context(max_devices=state['batch_size'])
         net_params = net.collect_params()
         _mxnet_utils.load_net_params_from_state(
             net_params, state['_model'], ctx=ctx 
             )
         state['_model'] = net
+        if state['_class_type'] == "int":
+            state['classes'] = list(map(int, state['classes']))
```

Comment:
We need this since when saved models are loaded, classes of type int are loaded as floats -- something that we believe is part of a deeper issue involving the `CustomModel` in `_model.py` that calls into `turicreate.connect.main`. I discussed this with @srikris last night so I'll let him chime in if I'm not articulating something properly. :-) Should I file a follow-up issue to get rid of this hack?

Reply:
Fixed.